src/app/clusters/barrier-control-server/barrier-control-server.cpp

```diff
@@ -51,6 +54,28 @@ static State state;
 #define ZCL_USING_BARRIER_CONTROL_CLUSTER_BARRIER_COMMAND_CLOSE_EVENTS_ATTRIBUTE
 #endif
 
+/**********************************************************
+ * Matter timer scheduling glue logic
+ *********************************************************/
+
+void emberAfBarrierControlClusterServerTickCallback(EndpointId endpoint);
+
+static void timerCallback(System::Layer *, void * callbackContext)
+{
+    emberAfBarrierControlClusterServerTickCallback(static_cast<EndpointId>(reinterpret_cast<uintptr_t>(callbackContext)));
+}
+
+static void schedule(EndpointId endpoint, uint32_t delayMs)
```

Comment:
I wonder about `schedule/deactivate` naming ... they don't seem obvious.

How about `scheduleTimerCallackMs` and `deactivateEndpointTimerCallback` ?

Reply:
renamed, ptal